### ISSUE TITLE
Cause Detail cover images fail to load on 301s

### ISSUE THREAD

**Issue by aaronschachter** [DoSomething/LetsDoThis-iOS#1022]
Sometimes the Cause cover images appear with the failed image on device -- haven't been able to replicate it in Simulator.  Inspecting a Cause object from the API indicates that Wordpress is storing the uploaded image URL in the database, since we made the domain change in #1018:
````
{ cause: 
	   { id: 16,
	     title: 'Animals',
	     tagline: 'Sometimes Simba or Bambi or Dory need your help.',
	     image_url: 'http://live-ltd-news.pantheon.io/wp-content/uploads/2016/03/animals.png' },
	  campaignsUrl: 'https://www.dosomething.org/api/v1/campaigns?term_ids=16&count=100',
	  rootTag: 21 }
````
Verified that the image URL exists, but whats causing the `NetworkImage` react component error are redirects:

> [tid:com.facebook.React.JavaScript] The operation couldn’t be completed. (NSURLErrorDomain error 301.)
> May 18 14:12:07  DoSomething[308] <Notice>: The operation couldn’t be completed. (NSURLErrorDomain error 301.)
> 2016-05-18 14:12:07.600 [info][tid:com.facebook.React.JavaScript] The operation couldn’t be completed. (NSURLErrorDomain error 301.)
> May 18 14:12:07  DoSomething[308] <Notice>: The operation couldn’t be completed. (NSURLErrorDomain error 301.)

**Comment by aaronschachter** [comment]
👾  https://github.com/facebook/react-native/issues/4940

**Comment by lkpttn** [comment]
Re-uploaded all the images so they point directly to `pantheonsite.io` which negates the 301s.

**Comment by aaronschachter** [comment]
Looks like this may be fixed in React native 0.26 -- https://github.com/facebook/react-native/releases/tag/v0.26.0

> Fix images not displaying with HTTP response 301 - 192ab66
